packages/crypto/src/__tests__/proof-tests.ts

```diff
@@ -0,0 +1,115 @@
+    deploymentLabel: "r2025.9.2",
+    enclaveApp: "signer",
+    owner: "tkhq",
+    createdAt: {
+      // 2025-09-16 21:24:01.448145+00
+      seconds: "1758057841",
+      nanos: "448145000",
+    },
+  };
+
+  test("should verify valid app proof signatures", () => {
+    expect(() => verifyAppProofSignature(testAppProof1)).not.toThrow();
+    expect(() => verifyAppProofSignature(testAppProof2)).not.toThrow();
```

Comment:
"not throwing" is certainly good, but what is the return value? I think we should have a stricter assertion (`toEqual`?)